docs/xls-75d-delegating-account-permissions/concepts/delegating-account-permissions.md

```diff
@@ -0,0 +1,89 @@
+---
+seo:
+    description: Common configurations for an XRP Ledger account.
```

Comment:
Could we update the description to clarify that this page is about delegating account permissions?